views/dataset/refresh_publications.templ

```diff
@@ -0,0 +1,13 @@
+package dataset
+
+import (
+	"github.com/ugent-library/biblio-backoffice/ctx"
+	"github.com/ugent-library/biblio-backoffice/models"
+)
+
+templ RefreshPublications(c *ctx.Ctx, dataset *models.Dataset, relatedPublications []*models.Publication) {
+	<div hx-swap-oob="innerHTML:#modals"></div>
+	<div hx-swap-oob="innerHTML:#publications-body">
+		@publicationsBody(c, dataset, relatedPublications)
+	</div>
+}
```

Comment:
Zie ReplaceModal templ view

Reply:
agreed but this doesn't work like that ReplaceModal..